Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthWalletExtensions.cs

```diff
@@ -34,9 +37,13 @@ public static IWeb3ServiceCollection UseWeb3AuthWallet(this IWeb3ServiceCollecti
         collection.AssertServiceNotBound<ISigner>();
         collection.AssertServiceNotBound<ITransactionExecutor>();
 
-        collection.AddSingleton<ISigner, ITransactionExecutor, ILifecycleParticipant, Web3AuthWallet>();
-
-        collection.AddSingleton<IWeb3AuthTransactionHandler, Web3AuthTransactionHandler>();
+        collection.AddSingleton<IWalletProvider, Web3AuthProvider>();
+        
+        collection.Replace(ServiceDescriptor.Singleton(typeof(AccountProvider), new AccountProvider()));
```

Comment:
Will we ever have to replace it? Why aren't we just adding it as a regular singleton?

Reply:
hmm yeah that's right, I was just following what I saw everywhere else, will fix

Reply:
done